products/public-websites/cerner/postmortem/missing-content-schedule-appt-page.md

```diff
@@ -0,0 +1,103 @@
+
+Date: 08/17/2020
+
+Authors: Kelson Adams
+
+Status: Resolved
+
+## What happened?
+
+In the morning of 08/17/2020, [a PR was merged](https://github.com/department-of-veterans-affairs/vets-website/pull/13882/files#diff-fbc145d9742763ba3cf52c0de9768e98R429) that updated legacy content on [www.va.gov/health-care/schedule-view-va-appointments](https://www.va.gov/health-care/schedule-view-va-appointments/) in response to [this request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12446#issuecomment-674324572).
+
+When the PR was deployed at 3:15pm EST, Danielle Thierry then discovered it at 4:05pm EST. (2:05pm MST = 4:05pm EST)
+
+![Discovery message](./discovery.png)
+
+Traffic was almost immediately observed to go down to 0 for VAOS:
+
+![Grafana](./grafana.png)
+
+When Danielle raised the issue to Kelson Adams, Kelson opened up the dev console on the page [www.va.gov/health-care/schedule-view-va-appointments](https://www.va.gov/health-care/schedule-view-va-appointments/) and found that there was a runtime error with the common `<Telephone />` component on that page where there was a missing prop called `pattern` that is _ONLY_ required for 3-digit phone numbers (in this case, it was TTY 711 numbers). A [PR was sent up to fix this](https://github.com/department-of-veterans-affairs/vets-website/pull/13896) 5 minutes later at 4:10pm EST. The PR was approved by VSP and merged at 5pm EST.
+
+An [off-schedule deploy was requested](https://dsva.slack.com/archives/CBU0KDSB1/p1597695472119100?thread_ts=1597695207.118200&cid=CBU0KDSB1) at 4:17pm EST, which was then executed at 5:52pm EST. Following the fix being deployed at that time, traffic resumed as normal to VAOS.
+
+## Background
+
+A request was made to update the current content on the page [www.va.gov/health-care/schedule-view-va-appointments](https://www.va.gov/health-care/schedule-view-va-appointments/) with [this GitHub comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12446#issuecomment-674324572).
+
+> we separately need to update the legacy content for this schedule and view appointments page for the Monday 8.17 deploy. The VA appointments tool is adding community care appointments that day, but we won't be quite ready to just switch to the unauthenticated Cerner content. I'm attaching the needed changes to the legacy page. Can you update these or show me how to update Monday morning? Thank you!
+
+[This request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12446#issuecomment-674324572)
+
+## Detection
+_How was the issue found?_ It was found by Danielle Thierry.
+
+![Discovery message](./discovery.png)
+
+_Date/time of discovery_ 4:05pm EST
+
+## Impact
+
+_What was the impact on Veterans, business and team?_
+
+All traffic to VAOS from [www.va.gov/health-care/schedule-view-va-appointments](https://www.va.gov/health-care/schedule-view-va-appointments/) ceased from 3:15pm EST to 5:52pm EST.
```

Comment:
```suggestion
Because `/health-care/schedule-view-va-appointments` serves as the gateway to VA Online Scheduling and was unavailable, traffic to VAOS from ceased from 3:15pm EST to 5:52pm EST.
```